### PR TITLE
Fix VVL_ENABLE_ASAN

### PR DESCRIPTION
**Pull request by juan-lunarg**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -111,10 +111,8 @@ endif()
 
 option(VVL_ENABLE_ASAN "Use address sanitization")
 if (VVL_ENABLE_ASAN)
-    if (MSVC)
-        add_compile_options(/fsanitize=address)
-    else()
-        add_compile_options(-fsanitize=address -fno-omit-frame-pointer)
+    add_compile_options(-fsanitize=address)
```
Comment (juan-lunarg) [inline CMakeLists.txt:114]: MSVC compiler accepts either `-` or `/` so I wrote the code to take that into account.

File: scripts/update_deps.py [hunk 2]
```diff
@@ -491,22 +491,10 @@ def CMakeConfig(self, repos):
...
-                cmake_c_flags = cmake_c_flags + ' /fsanitize=address'
-            else:
-                cmake_cxx_flags = cmake_cxx_flags + ' -fsanitize=address -fno-omit-frame-pointer'
-                cmake_c_flags = cmake_c_flags + ' -fsanitize=address -fno-omit-frame-pointer'
-            cmake_cxx_flags = cmake_cxx_flags.strip()
-            cmake_c_flags = cmake_c_flags.strip()
-            cmake_cmd.append(f'-DCMAKE_CXX_FLAGS="{cmake_cxx_flags}" -DCMAKE_C_FLAGS="{cmake_c_flags}"')
-
+            cmake_cmd.append(f'-D CMAKE_CXX_FLAGS=-fsanitize=address')
+            cmake_cmd.append(f'-D CMAKE_C_FLAGS=-fsanitize=address')
```
Comment (artem-lunarg) [inline scripts/update_deps.py:495]: Previous version added spaces between commands (`' /fsanitize=address'`. Do we need space before `-D` here?

Comment (juan-lunarg) [reply scripts/update_deps.py:495]: > Do we need space before -D here?

The space between the -D is consistent with what is on line 490. Helps with readability IMO as well.

**Review by artem-lunarg — APPROVED** [review]

**Review by spencer-lunarg — APPROVED** [review]

File: scripts/update_deps.py (hunk 2 above)
Comment (arno-lunarg) [reply scripts/update_deps.py:495]: I wrote this convoluted logic thinking that going just for `-D CMAKE_CXX_FLAGS=-fsanitize=address` would not append to but overwrite `CMAKE_CXX_FLAGS`. So does this change really appends the sanitize flag?

**Review by arno-lunarg — APPROVED** [review]

**Comment by juan-lunarg** [comment]
> So does this change really appends the sanitize flag?

It will just append. Feel free to try it out locally.

**Pull request merged**